Yeoman generator not working on windows correctly

Langium version: 0.1.0-next.00ebfd9
Package name: generator-langium

## Steps To Reproduce

1. Run the yeoman generator for langium on a **Windows** machine.
2. Observe the error message.

## The current behavior

Running the generator currently results in the following error message:

```
npm ERR! code ENOENT
npm ERR! syscall open
npm ERR! path C:\Users\Mark\Source\Repos\langium-test/package.json
npm ERR! errno -4058
npm ERR! enoent ENOENT: no such file or directory, open 'C:\Users\User\Source\Repos\langium-test\package.json'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\User\AppData\Local\npm-cache\_logs\2021-06-16T13_04_30_619Z-debug.log
Error langium

Command failed with exit code 4294963238: npm install --prefix C:\Users\User\Source\Repos\langium-test\hello-world
```

## The expected behavior

No error message and everything working smoothly ;)

--- a/packages/generator-langium/src/index.ts
+++ b/packages/generator-langium/src/index.ts
@@ -1,5 +1,5 @@
 import Generator from 'yeoman-generator';
-import * as _ from 'lodash';
+import _ from 'lodash';
 import * as path from 'path';
 
 const TEMPLATE_DIR = '../langium-template';
@@ -141,9 +141,10 @@ class LangiumGenerator extends Generator {
             this.answers.extensionName
         );
 
-        this.spawnCommandSync('npm', ['install', '--prefix', extensionPath]);
-        this.spawnCommandSync('npm', ['run', '--prefix', extensionPath, 'langium:generate']);
-        this.spawnCommandSync('npm', ['run', '--prefix', extensionPath, 'build']);
+        const opts = { cwd: extensionPath };
+        this.spawnCommandSync('npm', ['install'], opts);
+        this.spawnCommandSync('npm', ['run', 'langium:generate'], opts);
+        this.spawnCommandSync('npm', ['run', 'build'], opts);
     }
 }
 

